newIDE/app/src/Profile/Team/TeamProvider.js

```diff
@@ -112,11 +112,17 @@ const TeamProvider = ({ children }: Props) => {
   const onCreateMembers = React.useCallback(
     async quantity => {
       if (!team || !profile) return;
-      await createTeamMembers(getAuthorizationHeader, {
+      const createdUsers = await createTeamMembers(getAuthorizationHeader, {
```

Comment:
should there be try catches around all the calls made in this file? to display an error in case it fails